README.md

```diff
@@ -55,5 +55,14 @@ services.AddTransient(typeof(IRequestHandler<,>), typeof(GenericHandlerBase<,>))
 ```
 
 This won't work with generic constraints, so you're better off creating an abstract base class and concrete closed generic classes that fill in the right types.
```

Comment:
I don't think this statement is true anymore? MS.Ext.DI supports generic constraints now.